libs/filer/workspace_files_client.go

```diff
@@ -115,7 +115,7 @@ func (w *WorkspaceFilesClient) Write(ctx context.Context, name string, reader io
 		return err
 	}
 
-	err = w.apiClient.Do(ctx, http.MethodPost, urlPath, body, nil)
+	err = w.apiClient.Do(ctx, http.MethodPost, urlPath, nil, body, nil)
```

Comment:
Do we need to specify the Content-Type header?

Reply:
We don't -- this is covered in integration tests.